infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -177,35 +179,41 @@ private void stopLogReplicationRuntime(String remoteClusterId) {
         }
     }
 
+    /**
+     * Update Log Replication Runtime config id.
+     */
+    public void updateRuntimeConfigId(TopologyDescriptor newConfig) {
+        runtimeToRemoteCluster.values().forEach(runtime -> runtime.updateFSMConfigId(newConfig));
+    }
+
     /**
      * The notification of change of adding/removing standby's without epoch change.
      *
      * @param newConfig has the same topologyConfigId as the current config
      */
     public void processStandbyChange(TopologyDescriptor newConfig) {
         if (newConfig.getTopologyConfigId() != topology.getTopologyConfigId()) {
-            log.error("Detected changes in the topology. The new topology descriptor {} doesn't have the same " +
+            log.warn("Detected changes in the topology. The new topology descriptor {} doesn't have the same " +
```

Comment:
We do not assume that config id change implies a role change, so it is possible that we have a higher config id.

Reply:
We may add more comments to the function to explain that this function only is executed if the configID doesn't change.

Reply:
Ideally, there is no config id change when we only processStandbyChange. I will add a short comment.